src/tests/dbus-tests/run_tests.py

```diff
@@ -53,6 +54,30 @@ def setup_vdevs():
     storagedtestcase.test_devs = vdevs
 
 
+def install_new_policy(projdir):
+    '''Copies the polkit policies to the system directory and backs up eventually the existing files.
+       Returns a list of tuples containing (file_name, should_be_restored).'''
+    files = glob.glob(projdir + '/data/*.policy') + glob.glob(projdir + '/modules/*/data/*.policy')
+    restore_list = []
+    for f in files:
+        tgt = '/usr/share/polkit-1/actions/' + os.path.basename(f)
+        if os.path.exists(tgt):
+            shutil.move(tgt, '/tmp/')
```

Comment:
Maybe, I would move the config files to `$TMPDIR/storaged` - so we don't spoil the `$TMPDIR`... and for `/tmp` I'd use `tempfile` module: `tempfile.gettempdir()`.